test/cypress/support/commands.js

```diff
@@ -167,9 +167,6 @@ Cypress.Commands.add('giCreateGroup', (name, {
     cy.getByDT('app').then(([el]) => {
       cy.get(el).should('have.attr', 'data-sync', '')
     })
-
-    checkIfJoinedGeneralChannel(name)
-
```

Comment:
Actually, I am not sure if this removal breaks cypress test...

Reply:
They passed though didn't they

Reply:
Yeap. Nice catch. It should not be here. It's for ByPassUI.